### PR TITLE
Add option to escape search state when in SubscriptionSearchScreen

### PR DESCRIPTION
**Pull request by nashvail**

### CODE REVIEW

**Comment by borisyankov** [comment]
That is a good fix. Thanks.
PR is ready to merge, but GitHub reports some weird conflict. Can you rebase, please?

**Comment by nashvail** [comment]
Rebased!

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (borisyankov) [inline jsconfig.json]: Not sure if intentional, but not a problem to add this VS file.

**Comment by borisyankov** [comment]
Looks good!

**Pull request merged**